index.md

```diff
@@ -331,35 +331,35 @@ Go to `System / SIP` and enter the information about your SIP Server. Make sure,
 
 ### Find out the IPEI of your PP
 
-As a prerequisit to the next step you have to know the `IPEI` of the PP you want to register:
+As a prerequisite to the next step you have to know the *IPEI* of the PP you want to register:
 - Some manufacturers print the IPEI on the PP (e.g. inside the battery tray).
-- If you can't find the IPEI, go to `SIP Users / Decives`, set `Subscription` to `Subscription` and apply with `OK`. Now go to `System > Event Log` and try to register your PP by using the PIN 0000. The registration will fail, however after reloading the website, we can now get the IPEI from the event log.
+- If you can't find the IPEI, go to `SIP Users / Devices`, set `Subscription` to `Subscription` and apply with `OK`. Now go to `System > Event Log` and try to register your PP by using the PIN 0000. The registration will fail, however after reloading the website, we can now get the IPEI from the event log.
 
 `GMI : AccessRightsFailed(ipei=XXXXX XXXXXXX X, Missing DECT_IE_PORTABLE_ID or no wildcard subscription)`
 
 ### Registering a PP
 
 Go to the `SIP Users/Devices` page. At the `Create a new SIP User/Device` line, click `New`. A new window will open that lets you add a new User-Phone-Combination. **TODO*: SIP acconts <> USER <> DEVICES ???*
 
-General Settings
+#### General Settings
 
-| Setting                  | Example         | Description                 |
-| ------------------------ | --------------- | --------------------------- |
-| Display Name             | 3000            | SIP Display Name (optional) |
-| Number/SIP user name     | 3000            | SIP Authentication User     |
-| User/Device relation     | fixed           | USER / Device relationship  |
-| IPEI                     | 12345 1234567 1 | IPEI of the PP to register  |
-| DECT authentication code | 0000            | DECT-PIN                    |
+| Setting                  | Example         | Description                                                                 |
+| ------------------------ | --------------- | --------------------------------------------------------------------------- |
+| Display Name             | 3000            | SIP display name (optional)                                                 |
+| Number/SIP user name     | 3000            | DECT-internal phone number; used as SIP username if not overridden below    |
```

Comment:
Actually, this is "as far as I know"… @ioeides Can you check whether this actually works (and SIP-DECT allows you to leave "SIP > Authentication user name" empty)?